src/datastore/postgres/PgDataStore.ts

```diff
@@ -16,7 +16,13 @@ limitations under the License.
 
```

Comment:
I made these changes to fix the lint, probably could have done with it's own PR but oh well.